content/_releases/22/44.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Product Update - v22.44
+description: Details of the product update v22.44 from 3rd of November 2022.
+layout: rel
+version: "22.44"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-11-03
+redirect_from:
+  - /releases/22.44.html
+---
+
+## Improvements
+
+### Component repositories: New design & improved functionality
+
+We continue to update our User Interface to improve ease of use. This time it is
```

Comment:
In this release we update the UI of the Component repository pages.. When you navigate to Developer Teams on the side menu and then select a Component repository you will notice the new design.